Client/mods/deathmatch/logic/CClientEffect.cpp

```diff
@@ -71,7 +71,7 @@ float CClientEffect::GetEffectSpeed() const
 
 bool CClientEffect::SetEffectDensity(float fDensity)
 {
-    if (fDensity >= 0)
+    if (fDensity >= 0.0f && fDensity <= 2.0f)
```

Comment:
Is `2.0f` available as a constant anywhere? Like `MAX_EFFECT_DENSITY` or something?

Reply:
I have not found any related constants.